### PR TITLE
How to guide for increasing the open file limit on macOS, Linux, and …

### LINKED ISSUE #74: Maxfile Descriptor limit error : Running CockroachDB using rancher desktop
I am seeing an error when I am trying to run cockroach DB image using rancher desktop

My docker-compose file

```
version: '3.5'

services:

 db-node-1:
    container_name: db-node-1
    image: cockroachdb/cockroach:latest
    volumes:
      - ./data/db-node-1:/cockroach/cockroach-data
    command: start-single-node --insecure
    ports:
      - '26257:26257'
      - '8080:8080'

```

command: `nerdctl compose up`

error:

```
db-node-1_1 |time="2022-05-13T20:33:35Z" level=fatal msg="no running task found: task 9125f9e13d04a92fce2045ae459d4659598eec1ecb28f02c5bfca54818a5f598 not found: not found"
db-node-1   |*
db-node-1   |* WARNING: ALL SECURITY CONTROLS HAVE BEEN DISABLED!
db-node-1   |* 
db-node-1   |* This mode is intended for non-production testing only.
db-node-1   |* 
db-node-1   |* In this mode:
db-node-1   |* - Your cluster is open to any client that can access any of your IP addresses.
db-node-1   |* - Intruders with access to your machine or network can observe client-server traffic.
db-node-1   |* - Intruders can log in without password and read or write any data in the cluster.
db-node-1   |* - Intruders can consume all your server's resources and cause unavailability.
db-node-1   |*
db-node-1   |*
db-node-1   |* INFO: To start a secure server without mandating TLS for clients,
db-node-1   |* consider --accept-sql-without-tls instead. For other options, see:
db-node-1   |* 
db-node-1   |* - https://go.crdb.dev/issue-v/53404/v21.2
db-node-1   |* - https://www.cockroachlabs.com/docs/v21.2/secure-a-cluster.html
db-node-1   |*
db-node-1   |*
db-node-1   |* ERROR: ERROR: failed to start server: failed to create engines: hard open file descriptor limit of 1024 is under the minimum required 1956
db-node-1   |* please see https://www.cockroachlabs.com/docs/v21.2/recommended-production-settings.html for more details
db-node-1   |*
db-node-1   |ERROR: failed to start server: failed to create engines: hard open file descriptor limit of 1024 is under the minimum required 1956
db-node-1   |please see https://www.cockroachlabs.com/docs/v21.2/recommended-production-settings.html for more details
db-node-1   |Failed running "start-single-node"

```

### PR DESCRIPTION
**Pull request by sunilarjun**
…Windows. Written for all versions and updated the versions sidebar as well. This PR fixes #74 .

### CODE REVIEW

File: docs/how-to-guides/increasing-open-file-limit.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    EOF
+```
+
+Lastly, please stop and restart Rancher Desktop in order for the updated limits to take effect.
+
+## Windows Steps
+
+First, be sure that you have ran Rancher Desktop at least once in order for the configurations to initialize.
+
+Users will then have access to the `%AppData%\rancher-desktop\provisioning` directory. Next, please create a file in this directory that will hold the script to update the open file limit by increasing the `max_map_count` inside the vm, ex: `%AppData%\rancher-desktop\provisioning\map_count.start`.
```
Comment (gunamata) [inline docs/how-to-guides/increasing-open-file-limit.md:67]:
```suggestion
You can then create a provisioning script, say `map_count.start`, at `%AppData%\rancher-desktop\provisioning` with the below code to update the open file limit by increasing the `max_map_count` parameter.
```

File: docs/how-to-guides/increasing-open-file-limit.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Implement the script below in your created `*.start` file.
```
Comment (gunamata) [inline docs/how-to-guides/increasing-open-file-limit.md:69]:
```suggestion
```

**Review by gunamata — CHANGES_REQUESTED** [review]
Proposed some changes to have a consistent tone (second person) and minor rephrasing to improve readability. If agree with the proposals then please apply the changes to other versions too.

**Comment by sunilarjun** [comment]
Updated all versions with recommended changes, thank you for the review Guna! If all looks good I can go ahead and merge the changes.

**Review by gunamata — APPROVED** [review]
LGTM, Thank you.

**Pull request merged**